moveit_plugins/moveit_ros_control_interface/src/joint_trajectory_controller_plugin.cpp

```diff
@@ -51,7 +51,7 @@ class JointTrajectoryControllerAllocator : public ControllerHandleAllocator
   virtual moveit_controller_manager::MoveItControllerHandlePtr alloc(const std::string &name,
                                                                      const std::vector<std::string> &resources)
   {
-    return boost::make_shared<moveit_simple_controller_manager::FollowJointTrajectoryControllerHandle>(
+    return std::make_shared<moveit_simple_controller_manager::FollowJointTrajectoryControllerHandle>(
```

Comment:
`<memory>` is not included here, but `<boost/shared_ptr.hpp>` still is. Is the latter still required here?